tests/unit/components/adminLevelToggle.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { render } from "@testing-library/vue";
+import { describe, expect, test } from "vitest";
+import { createVuetify } from "vuetify";
+import { mockPinia } from "../mocks/mockPinia";
+import AdminLevelToggle from "../../../src/components/AdminLevelToggle.vue";
+
+const store = mockPinia({ selectedAdminLevel: 1 });
+const renderComponent = () => {
+    const vuetify = createVuetify();
+
+    return render(AdminLevelToggle, {
+        global: {
+            plugins: [store, vuetify]
+        }
+    });
+};
+
+describe("AdminLevelToggle", () => {
+    test("renders as expected", async () => {
+        const { findAllByRole } = renderComponent();
+        const buttons = await findAllByRole("link");
+        expect(buttons.length).toBe(2);
+        const [button1, button2] = buttons;
+        expect(button1).toHaveClass("mb-2 rounded-s-lg rounded-e-0 toggle-btn bg-blue");
```

Comment:
I'm not used to such specific tests as this. What's the value in testing for classes?

Reply:
yep this was too specific, have removed it